lib/ansible/plugins/callback/debug.py

```diff
@@ -0,0 +1,45 @@
+import codecs
+import imp
+import os
+
+# For some reason this would cause ImportError
+# import ansible.plugins.callback.default as DEFAULT_MODULE
+# So, please forgive the horror you are about to witness, but it is necessary
```

Comment:
^ the code above is what we already use in other plugins that derive from default